test/cpp/tensorexpr/test_external_calls.cpp

```diff
@@ -237,5 +237,91 @@ TEST(ExternalCall, Matmul) {
   ASSERT_TRUE(at::allclose(nnc_result, ref));
 }
 
+TEST(ExternalCall, SeveralCallsUsedInEachOther) {
```

Comment:
I'd like a more concise and expressive name for this test, such as `ExpressionArguments` or something similar.

Reply:
How about `ComputeInterop` to underline that the test is checking how external-calls tensors interoperate with tensors built with the Compute API? I also added a comment on that.

Reply:
It's ok.